src/Resources/views/tailwind_2/_header.html.twig

```diff
@@ -106,15 +108,14 @@
                                 value="{{ table.searchExtension.query }}"
                             >
                         </div>
-                        <button class="whatwedo_table-button--search" type="submit">
-                            {{ bootstrap_icon('search', { class: 'h-4 w-4 text-neutral-400', alt: 'Suchen' }) }}
+                        <button class="whatwedo_table-button--search" type="submit" title="Suche">
```

Comment:
Let's fix that button altogether:

1. Add an `sr-only` Text to the button. Buttons either have an `aria-label` or a text, where the former is a last resort to use.
2.  Keep the title for mouse users too, but make it translatable.